toolkit/src/components/cut/list-item/action.hbs

```diff
@@ -0,0 +1,11 @@
+{{!
+  Copyright (c) HashiCorp, Inc.
+}}
+
+<div {{on 'click' this.onClickStopPropagation}}>
```

Comment:
adding attributes spreading can be useful to add custom IDs or class names, or `data/test-data` attributes:
```suggestion
<div {{on 'click' this.onClickStopPropagation}} ...attributes>
```